crates/bevy_sprite_render/src/mesh2d/material.rs

```diff
@@ -737,7 +737,10 @@ pub fn specialize_material2d_meshes<M: Material2d>(
             let Some(mesh_instance) = render_mesh_instances.get_mut(visible_entity) else {
                 continue;
             };
-            let entity_tick = entity_specialization_ticks.get(visible_entity).unwrap();
+            let Some(entity_tick) = entity_specialization_ticks.get(visible_entity) else {
+                error!("{} is missing specialization tick. Spawning Mesh2d in PostUpdate or later is currently not fully supported. Please see PR #19064 for details.", visible_entity);
```

Comment:
This isn't just for `Mesh2d`, it also affects `Meah`.  Also, let's not link to the specific PR as we're likely not going to merge that one in particular.